src/composables/PluginManager.js

```diff
@@ -227,15 +227,23 @@ export async function initComponents(projectName, plugin, path) {
  * @param {Object} plugin - Plugin corresponding to the model.
  * @param {String} path - Model path (Plugin name & model name).
  * @param {Object} definition - Definition of the component.
+ * @param {ComponentDrawOption} [drawOption=null] - drawOption of the component.
```

Comment:
no need to import `ComponentDrawOption` ?

Reply:
We pass the parameter into another file, and it works like this, if I import `ComponentDrawOption` the linter tells me that it is not used.

Reply:
my bad, it's only the doc for the param, no need to import anyways